src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -477,7 +477,13 @@ function ($voucher) {
                     'Shop.Theme.Checkout'
                 ) :
                 '',
+        ];
+
+        Hook::exec('overridePresentedCart',
```

Comment:
What about https://devdocs.prestashop.com/1.7/modules/concepts/hooks/#naming-scheme :thinking: 
```suggestion
        Hook::exec('actionPresentCart',
```
wdyt?

Reply:
I saw a 3rd type of hooks here: https://github.com/PrestaShop/PrestaShop/blob/develop/src/Adapter/Presenter/Cart/CartPresenter.php#L423 and it looked good 🤔but I can change the PR if it's better

Reply:
Or update the doc :thinking: